docs/learn/smart-contract-developers/mint-lsp7-token.md

```diff
@@ -8,196 +8,90 @@ import TabItem from '@theme/TabItem';
-  <TabItem value="web3js" label="web3.js">
-
-```javascript
-const myToken = new web3.eth.Contract(LSP7Mintable.abi, myTokenAddress);
+// setup signer
+const signer = new ethers.Wallet(privateKey as string, provider);
+console.log('🔑 EOA: ', signer.address);
 ```
 
-  </TabItem>
+### Instantiate the contracts
 
-  <TabItem value="ethersjs" label="ethers.js">
+At this point, the `LPS7Mintable` contract is being prepared for the following intercation. you construct an instance of a contract, using _contract ABI_ and _contract address_.
 
 ```javascript
-const myToken = new ethers.Contract(myTokenAddress, LSP7Mintable.abi);
+let myToken = new ethers.Contract(myTokenAddress, LSP7Mintable.abi, signer);
 ```
 
-  </TabItem>
+### Send transaction
 
-</Tabs>
-
-### Step 3 - Send transaction
-
-Finally, we send the transaction and mint some tokens to the Universal Profile of your choosing.
+Finally, you send the transaction and mint some tokens.
 
 :::warning
 
-The contract that we are using as a example in this guied allows minting Digital Assets **only to the owner** of the contract. There might be contracts that don't have this _requirement_.
+The example contract here allows minting Digital Assets **only to the owner** of the contract. There might be contracts that don't have this _requirement_.
 
 :::
 
-<Tabs>
-  
-  <TabItem value="web3js" label="web3.js">
-
-<!-- prettier-ignore-start -->
-
 ```javascript
-await myToken.methods.mint(universalProfileAddress, 100, false, '0x').send({ from: myEOA.address, gas: 100_000 });
+let mintTxn = await myToken.mint(signer.address, 1, true, '0x', {
+  gasLimit: 400_000,
+});
 ```
 
-<!-- prettier-ignore-end -->
-
-  </TabItem>
-
-  <TabItem value="ethersjs" label="ethers.js">
-
-```javascript
-await myToken.connect(myEOA).mint(universalProfileAddress, 100, false, '0x');
-```
-
-  </TabItem>
-
-</Tabs>
-
 ### Final code
 
-<Tabs>
-  
-  <TabItem value="web3js" label="web3.js">
-
-<!-- prettier-ignore-start -->
-
 ```javascript
+import { ethers } from 'hardhat';
 import LSP7Mintable from '@lukso/lsp-smart-contracts/artifacts/LSP7Mintable.json';
-import Web3 from 'web3';
-
-const web3 = new Web3('https://rpc.testnet.lukso.network');
-const privateKey = '0x...';
-const myTokenAddress = '0x...';
-const universalProfileAddress = '0x...';
 
-// setup your EOA
-const myEOA = web3.eth.accounts.wallet.add(privateKey);
+const privateKey = '0x..';
+const myTokenAddress = '0x..';
 
-const myToken = new web3.eth.Contract(LSP7Mintable.abi, myTokenAddress);
+async function main() {
 
-await myToken.methods.mint(universalProfileAddress, 100, false, '0x').send({ from: myEOA, gas: 100_000 });
-```
+  const provider = new ethers.JsonRpcProvider(
+    'https://rpc.testnet.lukso.network',
+  );
 
-<!-- prettier-ignore-end -->
+  const signer = new ethers.Wallet(privateKey as string, provider);
+  console.log('🔑 EOA: ', signer.address);
 
-  </TabItem>
+  let myToken = new ethers.Contract(myTokenAddress, LSP7Mintable.abi, signer);
```

Comment:
Suggest const since variable is unchanged afterwards

```suggestion
  const myToken = new ethers.Contract(myTokenAddress, LSP7Mintable.abi, signer);
```